metric_learn/itml.py

```diff
@@ -192,7 +196,7 @@ def fit(self, pairs, y, bounds=None, calibration_params=None):
     calibration_params = (calibration_params if calibration_params is not
                           None else dict())
     self._validate_calibration_params(**calibration_params)
-    self._fit(pairs, y)
+    self._fit(pairs, y, bounds=bounds)
```

Comment:
This was missing before, but the new tests in this PR made it pop up that it's needed to transmit the bounds argument, otherwise no matter what it will be default (None)